Amasty_Storelocator/styles/modules/_store-locator.scss

```diff
@@ -91,6 +91,40 @@
     .input__field {
         font-size: $font-size-base;
     }
+    // to overwrite amasty styles (load in php :/)
+    &.amlocator-main-container {
+        .amlocator-button {
+            color: $white;
+            background-color: $color-primary;
+            border: $border-width-base $border-style-base $color-primary;
```

Comment:
Optionaly:
I guess it could be good to move some variables to module level. I mean create  `$store-locator__border: $border-width-base $border-style-base $color-primary` etc. as it is easier later just to override module variables instead of whole while. At leas those variables that probably will be mostly modified etc.

Reply:
yup, you're right, I'll fix it